### PR TITLE
[Doc Enhancement][Localize-your-app-update][2267329]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ccf8435ca6454bcbfc7a8f3bb50e5068f9b5f104/msteams-platform/concepts/build-and-test/apps-localization.md) | 82 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1fb100f8-e539-436c-86e9-a9a35124d62c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/apps-localization.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Consider the following factors to localize your Microsoft Teams app:
...
+If you're publishing the app to the store, provide metadata (descriptions, screen shots, name) in the languages that you would like to be listed in, and explicitly specify these languages on the Marketplace listings page in Partner Center. To support localized listings in the app store, you can add additional languages to your listing. The default language information you provide in [Partner Center](/office/dev/store/submit-to-appsource-via-partner-center) for your listing appears in the [AppSource website](https://appsource.microsoft.com/marketplace/apps?product=office%3Bteams&page=1 "AppSource is one place for all your team needs. bring everything together including chats, meetings, calls, files, and tools to enable more productive teamwork.") listing for your app. Currently, the default language is English.
```
Comment (v-habibaumme) [inline msteams-platform/concepts/build-and-test/apps-localization.md:19]: Partner Center - change to lower case

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/939289a8f3a027b00c1abc767c5b669e6d652b4b/msteams-platform/concepts/build-and-test/apps-localization.md) | 82 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e332379-f0ac-4596-a02c-dcca177b0724) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cf800334d4549b4fb201d41d122130d864e813f4/msteams-platform/concepts/build-and-test/apps-localization.md) | 82 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e6528bf6-cd9f-4812-be16-f1d90b9c7f6a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a74c28e0f609938611b1b658bc294d0e8ec9b176/msteams-platform/concepts/build-and-test/apps-localization.md) | 83 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d4f5c27c-cf6b-4e1d-a875-4fdb40b2de87) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5ea324e509988b0f21ac108613066b09f6feb4e9/msteams-platform/concepts/build-and-test/apps-localization.md) | 82 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b18fc155-8b71-4de0-8880-cee7ebc4100e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/39688addd441216e858d492ea2ea5a3f689329b4/msteams-platform/concepts/build-and-test/apps-localization.md) | 83 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eb5251f1-5fac-4e08-8021-5e627cef9ac9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]
Approve

**Pull request merged**